PocketKit/Sources/PocketKit/Collections/CollectionStoryViewModel.swift

```diff
@@ -72,15 +72,15 @@ extension CollectionStoryViewModel: RecommendationCellViewModel {
     }
 
     var excerpt: Markdown {
-        story.excerpt
+        storyModel.excerpt
     }
 
     var domain: String? {
-        story.publisher
+        storyModel.publisher
```

Comment:
seems that `nil` or empty string ruins the layout. What about making this a non-optional value?

Reply:
because `publisher` can be nil from the backend, we might want to fix cell constraints instead. We can file an issue on Jira